pkg/client/unversioned/extensions.go

```diff
@@ -44,10 +43,6 @@ type ExtensionsClient struct {
 	*RESTClient
 }
 
-func (c *ExtensionsClient) HorizontalPodAutoscalers(namespace string) HorizontalPodAutoscalerInterface {
```

Comment:
Not changing as explained in https://github.com/kubernetes/kubernetes/pull/20501#issuecomment-180260757